web/spacewalk-web.changes

```diff
@@ -1,3 +1,4 @@
+- Detailled build requirements.
```

Comment:
Nor relevant for customers. And TBH I wonder if the previous entry is relevant either.